contrib/test/integration/main.yml

```diff
@@ -89,6 +89,11 @@
     - name: clone build and install crun
       include: "build/crun.yml"
       when: "build_crun | default(False) | bool"
+
+    - name: clone build and install runc
+      include: "build/runc.yml"
+      when: "build_runc | default(True) | bool"
```

Comment:
@giuseppe Is it correct that `default(True)` only applies if `build_runc` is not set? That would mean in our case that it is always set becuse of vars.yml: `build_runc: False`.

Reply:
I guess so :)  we can probably drop that

Reply:
Ah okay I'll do.